src/main/convert.js

```diff
@@ -107,17 +107,17 @@ export class ThriftFileConverter {
     `export type ${this.transformName(def.id.name)} = ${this.types.convert(def.valueType)};`;
 
   generateEnumMap = (def: Enum) => {
-    const header = `{`;
+    const header = '{';
     const values = def.definitions
       .map((d, index) => `  "${d.id.name}": ${d.value ? d.value.value : index},`)
       .join('\n');
-    const footer = `}`;
+    const footer = '}';
 
     return [header, values, footer].join('\n');
   };
 
   generateEnum = (def: Enum) => {
-    return `export const ${this.transformName(def.id.name)} = ${this.generateEnumMap(def)};`;
+    return `export type ${this.transformName(def.id.name)} = ${this.generateEnumMap(def)};`;
```

Comment:
If the idea is that only the keys will ever be consumed, then changing `generateEnum` to generate out a union type instead of a map object makes more sense.